roles/template_service_broker/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
 # placeholder file?
 template_service_broker_remove: False
-template_service_broker_install: False
+template_service_broker_install: True
```

Comment:
Should it be `True` or `true`? `roles/ansible_service_broker/defaults/main.yml` uses `true`. If both work okay.

Reply:
Both are valid.  I prefer 'True' because it matches python's case.